vectorbender.py

```diff
@@ -103,28 +105,25 @@ def determineTransformationType(self):
             0 if no pairs Found
             1 if one pair found => translation
             2 if two pairs found => linear
-            3 if two pairs found => linear
-            4 if three or more pairs found => bending
-            5 if bending but unmet dependencies"""
+            3 if three or more pairs found => bending
+            4 if bending but unmet dependencies"""
 
         pairsLayer = self.dlg.pairsLayer()
 
         if pairsLayer is None:
             return 0
 
-        featuresCount = len(pairsLayer.selectedFeaturesIds()) if self.dlg.restrictBox_pairsLayer.isChecked() else len(pairsLayer.allFeatureIds())
+        featuresCount = len(pairsLayer.selectedFeatureIds()) if self.dlg.restrictBox_pairsLayer.isChecked() else len(pairsLayer.allFeatureIds())
         
         if featuresCount == 1:
             return 1
         elif featuresCount == 2:
             return 2
-        elif featuresCount == 3:
```

Comment:
same as above, we must keep the functionality for affine transform

Reply:
Done!